NightwarpComputers/Views/Routes/Submitted.cshtml

```diff
@@ -4,7 +4,7 @@
             <h2 class="text-success">Order has been submitted successfully!</h2>
         </div>
         <div class="row justify-content-center">
-            <p>Order ID: <b><i>{{orderId}}</i></b> has been receieved. You will be receiving a confirmation email shortly at: <b>{{email}}</b> from "NightwarpComputers@outlook.com.</p>
```

Comment:
Are you not using angularjs anymore?

Reply:
I am, I just found it easiest on this page to use the parameters I passed to my ASP RouteController in the ViewBag and access the values via Razor in the html. Although I do have the parameters in the url on this page, so I could grab them from my AngularJS script.

Reply:
okay cool, I was just wondering